linux_os/guide/system/permissions/partitions/mount_option_var_log_audit_nodev/rule.yml

```diff
@@ -0,0 +1,26 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Add nodev Option to /var/log/audit'
+
+description: |-
+    The <tt>nodev</tt> mount option can be used to prevent device files from
+    being created in <tt>/var/log/audit</tt>.
+    Legitimate character and block devices should exist only in
+    the <tt>/dev</tt> directory on the root partition or within chroot
+    jails built for system services.
+    {{{ describe_mount(option="nodev", part="/var/log/audit") }}}
+
+rationale: |-
+    The only legitimate location for device files is the <tt>/dev</tt> directory
+    located on the root partition. The only exception to this is chroot jails.
+
+severity: unknown
+
+identifiers:
+    cce@rhel8: 81095-2
+
+references:
```

Comment:
Add nist references to ``CM-7`` and ``MP-2``

CM-7 -- operate system with least privilege. Doesn't need nodev to operate

MP-2  -- multi media policy around removable drives